deploy-templates/prerequisites.env.example

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "ParameterKey": "AWSVpcId",
```

Comment:
Does this support a "Description" attribute that can be used to tell users where to get the value and an example of what the value would look like?

Reply:
Unfortunately not. Even if it's valid JSON, the AWS validator will only accept certain keys.